bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/connector/EpsonProjectorSerialConnector.java

```diff
@@ -83,7 +83,7 @@ public void connect() throws EpsonProjectorException {
                 this.out = out;
             }
         } catch (PortInUseException | UnsupportedCommOperationException | IOException e) {
-            throw new EpsonProjectorException(e);
+            throw new EpsonProjectorException(e.getMessage(), e);
```

Comment:
This seems redundant. The exception already contains the message.

Reply:
Removed